pkg/mounter/refcounter_windows.go

```diff
@@ -92,13 +91,13 @@ func incementRemotePathReferencesCount(mappingPath, remotePath string) error {
 
 // decrementRemotePathReferencesCount - removes reference between mappingPath and remotePath.
 // See incementRemotePathReferencesCount to understand how references work.
-func decrementRemotePathReferencesCount(mappingPath, remotePath string) error {
+func decrementRemotePathReferencesCount(mappingPath, remotePath string, volumeID string) error {
```

Comment:
remove `remotePath` parameter since it is not used in this function,

Reply:
also change function name `getRemotePathReferencesCount` --> `getVolumeIDReferencesCount`

Reply:
done